### DISCUSSION TITLE
Postal server on Azure (Port25 blocked)

### DISCUSSION THREAD

**Discussion by alex-pommier** [postalserver/postal#3006] [Help with using Postal]
I've read through various threads regarding port 25, 2525 & 587 am looking for a bit of clarity. My Azure subscription has been updated and now port 25 is closed.... As an alternative i'm thinking about sending mail outbound via 587 with TLS. As I understand it the server to server connections are normally initiated on port 25 and then based on handshake switch to 587 with TLS post successful handshake between the two servers.

If that is the case then im SOL. However i'm not sure of this. My requirements differer from many other posts i've read through in that I have a seperate email server NOT on azure that's all setup and working fine and it's the server that will be accepting incoming mail back for the addresses i'm going to be sending mail from.

So postal would only be responsible for sending mail outbound to recipients and have no need for it to receive mail and forward to my mailserver. I'm connecting to postal via API using MailWizz so connecting to Postal is a non-issue I believe as it's all going through the API and port 80 is open. It's just a matter of postal connecting to recipient servers via port 587 without needing port 25 open. Additionally tracking bounces, opens and other metrics displayed on postal i'd love to maintain and i'm not sure about this with regards to the ports. Can anyone offer some insight and educate me please & thank you!

**Comment by willpower232** [comment]
For what it is worth, I believe that having port 25 open is part of checks that receiving email servers do to ensure that your email server is capable of receiving any error message so I imagine if you can't have this open wherever you ended up, it wouldn't work out.

**Reply by alex-pommier** [reply]
Hi @willpower232 
Thanks for your reply, I closed the discussion as I found it's not doable server-to-server communication needs port 25 so im SOL. I'll have to find a different SP.
